images/ubuntu/toolsets/toolset-2204.json

```diff
@@ -289,8 +289,7 @@
         {"name": "fastlane"}
     ],
     "selenium": {
-        "version": "4",
-        "binary_name": "selenium-server"
```

Comment:
Should we really remove this key-value pair? It is used in more than 1 place and, should we need to change something, seems easier to manage

Reply:
I doubt it will ever change. So I consider this as redundant complexity that we might want to rid off. I can rollback it if you think it's better to keep this value.